packages/cache/src/index.ts

```diff
@@ -0,0 +1,161 @@
+import { Redis } from 'ioredis';
+import { LRUCache } from 'lru-cache';
+import { logger } from '@prairielearn/logger';
+import * as Sentry from '@prairielearn/sentry';
+
+const CACHE_KEY_PREFIX = 'prairielearn-cache:';
```

Comment:
Let's make this configurable in the `init` call. I think we should make this mandatory config to force callers to think about cache namespaces.

Reply:
Added `cacheKeyPrefix` to the `init` call and added this as a field in the configs so it can be set there.